### PR TITLE
refactor: changed some names for clarity

### PR DESCRIPTION
**Pull request by tdejager**
Refactor some names so that they are a bit less confusing.

### CODE REVIEW

File: src/cli/mod.rs [hunk 1]
```diff
@@ -71,6 +71,30 @@ pub enum Command {
     Remove(remove::Args),
 }
 
+#[derive(Parser, Debug, Default)]
+#[group(multiple = false)]
+/// Lock file usage from the CLI
+pub struct LockFileUsageArgs {
+    /// Don't update and check lock file
```
Comment (ruben-arts) [inline src/cli/mod.rs:78]:
```suggestion
    /// Don't check or update the lockfile, continue with previously installed environment.
```

File: src/cli/mod.rs [hunk 2]
```diff
@@ -71,6 +71,30 @@ pub enum Command {
...
+    #[clap(long, conflicts_with = "locked")]
+    pub frozen: bool,
+    /// Check if lock file is up to date
```
Comment (ruben-arts) [inline src/cli/mod.rs:81]:
```suggestion
    /// Check if lockfile is up to date, aborts when lockfile isn't up to date with the manifest file. 
```

**Review by ruben-arts — APPROVED** [review]

File: tests/common/mod.rs [hunk 3]
```diff
@@ -240,7 +241,8 @@ impl PixiControl {
         let project = self.project()?;
         let task = ExecutableTask::from_cmd_args(&project, args.task, Some(Platform::current()));
 
-        let task_env = get_task_env(&project, args.frozen, args.locked).await?;
+        let project = self.project().unwrap();
```
Comment (baszalmstra) [inline tests/common/mod.rs:244]: Isnt the project already availabe two lines above?

**Review by baszalmstra — COMMENTED** [review]
Great refactor!

File: tests/common/mod.rs (hunk 3 above)
Comment (tdejager) [reply tests/common/mod.rs:244]: You are right that was a merge error!

**Pull request merged**